checks/container_rt.go

```diff
@@ -36,9 +37,9 @@ func (r *RTContainerCheck) RealTime() bool { return true }
 
 // Run runs the real-time container check getting container-level stats from the Cgroups and Docker APIs.
 func (r *RTContainerCheck) Run(cfg *config.AgentConfig, groupID int32) ([]model.MessageBody, error) {
-	containers, err := container.GetContainers()
-	if err != nil && err != docker.ErrDockerNotAvailable {
-		return nil, err
+	containers, errs := container.GetContainers()
+	if len(errs) != 0 {
+		return nil, errors.New("failed to retrieve list of containers")
```

Comment:
I'm not sure this error will provide much more information besides what's already been logged in `GetContainers`. What if it didn't log but you instead used `errors.Wrap` (https://godoc.org/github.com/pkg/errors#hdr-Adding_context_to_an_error)?